Lib/test/test_httpservers.py

```diff
@@ -8,6 +8,7 @@
      SimpleHTTPRequestHandler, CGIHTTPRequestHandler
 from http import server, HTTPStatus
 
+import http.server
 import os
```

Comment:
better to keep the order for imports
```suggestion
import contextlib
import os
```